src/panels/lovelace/cards/hui-entity-button-card.ts

```diff
@@ -202,10 +202,10 @@ class HuiEntityButtonCard extends LitElement implements LovelaceCard {
   }
 
   private _computeColor(stateObj: HassEntity | LightEntity): string {
-    if (!stateObj.attributes.hs_color) {
+    if (!this._config!.icon_color && !stateObj.attributes.hs_color) {
       return "";
     }
-    const [hue, sat] = stateObj.attributes.hs_color;
+    const [hue, sat] = this._config!.icon_color || stateObj.attributes.hs_color;
```

Comment:
if `icon_color` is defined we can just return that instead of calculating from a `hs`?
That allows the user to give all color options:
- `#f0f0f0`
- `red`
- `rgba(123, 114, 122, .5)`
- etc
```js
    if (this._config!.icon_color) {
      return this._config!.icon_color;
    }
```

Reply:
good call